opentreemap/treemap/views/user.py

```diff
@@ -208,3 +208,34 @@ def user(request, username):
             'public_fields': public_fields,
             'private_fields': private_fields,
             'favorites': favorites}
+
+
+def users(request, instance):
+    max_items = request.GET.get('max_items', None)
+    query = request.GET.get('q', None)
+
+    users_qs = InstanceUser.objects \
+                           .filter(instance=instance)\
+                           .order_by('user__username')\
+                           .values('user_id', 'user__username',
+                                   'user__first_name', 'user__last_name',
+                                   'user__make_info_public')
+
+    if query:
+        users_qs = users_qs.filter(user__username__icontains=query)
+
+    if max_items:
+        users_qs = users_qs[:int(max_items)]
+
```

Comment:
Whether or not to accept this depends on how much we expect it to get used on maps with tens of thousands of users. Maybe we need input from @dboyer.

If there are many results, but the search string is late in the alphabet, the matches that start with the search string (and therefore are best matches) are likely to be clipped from the result by `max_items`.

Having used many search boxes in other web apps that do exactly that, I can attest that it is annoying. But if we regard this feature as fringe, maybe we don't care.

@dboyer, do we care?